src/main/webapp/app/exercises/shared/unreferenced-feedback/unreferenced-feedback.component.html

```diff
@@ -3,25 +3,31 @@
         <button id="add-unreferenced-feedback" class="btn btn-success mt-4" (click)="addUnreferencedFeedback()" [disabled]="readOnly">
             {{ 'artemisApp.fileUploadAssessment.addFeedback' | artemisTranslate }}
         </button>
-        <ng-container *ngIf="!busy">
```

Comment:
Note for code reviewers: The `busy` property was never set to `true`, so I removed this.

Reply:
Only the part with `<div *ngFor="let suggestion of feedbackSuggestions" class="col-12 col-lg-6 col-xl-6">` (line 22+) is new.